src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -145,6 +145,8 @@ The .NET Foundation licenses this file to you under the MIT license.
       <!-- No OpenSSL on Apple platforms or Android -->
       <NetCoreAppNativeLibrary Include="System.Security.Cryptography.Native.OpenSsl" Condition="'$(StaticOpenSslLinking)' != 'true' and '$(_IsApplePlatform)' != 'true' and '$(_targetOS)' != 'android'" />
       <NetCoreAppNativeLibrary Include="System.Security.Cryptography.Native.Android" Condition="'$(_targetOS)' == 'android'" />
+      <NetCoreAppNativeLibrary Include="System.Security.Cryptography.Native.Android.JNIExport" Condition="'$(_targetOS)' == 'android'" />
+      <IlcArg Include="--export-dynamic-symbol:Java_net_dot_android_crypto_DotnetProxyTrustManager_verifyRemoteCertificate" Condition="'$(_targetOS)' == 'android'" />
```

Comment:
```suggestion
    </ItemGroup>
    
    </ItemGroup Condition="'$(_targetOS)' == 'android'">
      <NetCoreAppNativeLibrary Include="System.Security.Cryptography.Native.Android" />
      <NetCoreAppNativeLibrary Include="System.Security.Cryptography.Native.Android.JNIExport" />
      <IlcArg Include="--export-dynamic-symbol:Java_net_dot_android_crypto_DotnetProxyTrustManager_verifyRemoteCertificate" />
```
Nit: This is quite a bit of specialization for Android crypto, it may deserve its own item group.